internal/dbusservice/internal_test.go

```diff
@@ -0,0 +1,99 @@
+package dbusservice
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/ubuntu/authd-oidc-brokers/conf"
+)
+
+var configTypes = map[string]string{
+	"valid": `
+[oidc]
+issuer = https://issuer.url.com
+client_id = client_id
+`,
+
+	"valid+optional": `
+[oidc]
+issuer = https://issuer.url.com
+client_id = client_id
+
+[users]
+home_base_dir = /home
+allowed_ssh_suffixes = @issuer.url.com
+`,
+
+	"template": conf.BrokerConfTemplate,
+}
+
+func TestParseConfig(t *testing.T) {
```

Comment:
This test is basically testing ini.Load() (apart from the last use case) :) 

My main issue is that everytime we update the config, we will need to update this test, so it could use golden files IMHO(marshalling in yaml for instance the maps to ensure the struct has been changed and loaded as we desired).